modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -314,11 +347,12 @@ Error GDScriptAnalyzer::resolve_inheritance(GDScriptParser::ClassNode *p_class,
 					return ERR_PARSE_ERROR;
 				}
 
-				Error err = info_parser->raise_status(GDScriptParserRef::INTERFACE_SOLVED);
+				Error err = info_parser->raise_status(GDScriptParserRef::INHERITANCE_SOLVED);
 				if (err != OK) {
 					push_error(vformat(R"(Could not resolve super class inheritance from "%s".)", name), p_class);
 					return err;
 				}
+				base = info_parser->get_parser()->head->get_datatype();
```

Comment:
Good eye catching this one! There have been a few issues with `autoloads` and this might be the source of at least a few of them?

Reply:
haha, yea, probably! i figured it was small enough that no one would care if it got fixed here 😄